requirements.txt

```diff
@@ -1,3 +1,5 @@
 git+https://github.com/TMiguelT/sphinxcontrib-restbuilder
 sphinx
 sphinx-autodoc-annotation
+nose
+pandas
```

Comment:
This should actually be `.`, which would make it inherit dependencies from the package itself (which is my mistake)